src/features/todo.js

```diff
@@ -0,0 +1,26 @@
+import { createSlice, nanoid } from "@reduxjs/toolkit";
+
+const initialState = [{id: nanoid(), task: "IAmHere", completed: false }]
+
+export const todoSlice = createSlice({
+    name: "todo",
+    initialState: {value: initialState},
+    reducers: {
+        addTodos : (state, action) => {
+            const todo = { id: nanoid(), task: action.payload, completed: false };
+            state.value =  [todo, ...state.value];
+          },
+        complete : (state, action) => {
+            const mapping = state.value.map((x) => x.id === action.payload ? { ...x, completed: !x.completed } : x); 
```

Comment:
The mapping argument is not really meaningful, improve it by finding a more meaningful name for the reader of your codes.